package.json

```diff
@@ -1,42 +1,62 @@
 {
   "name": "taskcluster-client",
-  "version": "1.2.1",
+  "version": "2.0.0",
   "author": "Jonas Finnemann Jensen <jopsen@gmail.com>",
   "description": "Client for interfacing taskcluster components",
   "license": "MPL-2.0",
+  "files": [
+    "build",
+    "apis.json"
+  ],
+  "main": "build/index.js",
+  "browser": "build/browser.js",
   "scripts": {
-    "test": "./test/runtests.sh"
-  },
-  "browser": "./browser.js",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/taskcluster/taskcluster-client.git"
+    "build:browser": "webpack",
+    "build:node": "babel src --out-dir build --source-maps inline --ignore node_modules",
+    "build": "npm run build:node && npm run build:browser",
+    "docs": "bin/docs.js",
+    "fetch": "bin/fetch.js",
+    "list": "bin/list.js",
+    "show": "bin/show.js",
+    "update": "npm run fetch && npm run build && npm run docs",
+    "test:node": "npm run build:node && mocha test/*_test.js",
+    "test:browser": "npm run build:browser && mocha-phantomjs -R spec --bail test/browser_test.html",
+    "test": "npm run test:node && npm run test:browser"
   },
+  "repository": "taskcluster/taskcluster-client",
   "dependencies": {
-    "lodash": "^3.6.0",
-    "debug": "^2.1.3",
-    "superagent": "~1.7.0",
-    "superagent-promise": "^0.2.0",
-    "superagent-hawk": "^0.0.6",
-    "amqplib": "^0.4.1",
-    "slugid": "^1.1.0",
-    "promise": "^6.1.0",
-    "hawk": "^2.3.1",
-    "url-join": "^0.0.1"
-  },
-  "optionalDependencies": {
-    "sockjs-client": "^1.0.3"
+    "amqplib": "0.4.2",
+    "assert": "1.4.1",
+    "debug": "2.2.0",
+    "hawk": "5.1.2",
+    "lodash.clonedeep": "4.5.0",
```

Comment:
Please do "^4.5.0" so we get the same version everywhere...
IMO using lodash is fine, but if you really only want clonedeep that's okay too... Because other packages depends on lodash and if we all use ^4 then we get the same version :)